extensions/adobe/experience/customerJourneyManagement/message-tracking-link.schema.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
The name is not very clear.
Should we just call it messageTracking or messageAction?